package.json

```diff
@@ -12,7 +12,7 @@
     "hoox": "0.0.1",
     "ip": "^1.1.5",
     "map-merge": "^1.1.0",
-    "multiserver": "^3.0.2",
+    "multiserver": "happy0/multiserver#22b5ff36a95062de58e22781788bae5039dbf2f0",
```

Comment:
@Happy0  Should this be changed to `^3.1.0`?

Reply:
I did that when I merged it.